app.js

```diff
@@ -44,9 +45,7 @@ app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
 
 // Use i18next
-app.use(i18midd.handle(i18next, {
-  removeLngFromUrl: false
-}));
+app.use(i18midd.handle(i18next));
```

Comment:
Are you suggesting this for some reason? The issue is solved just with the `cookieDomain` being correctly set.

Reply:
false is default: https://github.com/i18next/i18next-express-middleware/blob/013af5ed30410bb5375375c7aa1fe6219c2b783a/src/index.js#L52